ios/engine/KMEI/KeymanEngine/Classes/Manager.swift

```diff
@@ -697,11 +697,20 @@ public class Manager: NSObject, UIGestureRecognizerDelegate {
       let data = try Data(contentsOf: path, options: .mappedIfSafe)
       let jsonResult = try JSONSerialization.jsonObject(with: data, options: .mutableLeaves)
       if let jsonResult = jsonResult as? [String:AnyObject] {
+        var version: String = "1.0"
+
+        if let info = jsonResult["info"] as? [String:AnyObject] {
+          if let versionEntry = info["version"] as? [String:AnyObject] {
+            if let description = versionEntry["description"] as? String {
+              version = description;
```

Comment:
Technically, we need to validate that the string is a dotted numeric (e.g. `1.0.1`) and fall back to `1.0` if not. But willing to let this go for now.